.circleci/config.yml

```diff
@@ -182,13 +194,19 @@ jobs:
       - run:
           name: Test certain bundles against function constructors
           command: npm run no-new-func
+
+  test-dist2:
+    docker:
+      - image: circleci/node:12.13.0
+    working_directory: ~/plotly.js
+    steps:
+      - attach_workspace:
+          at: ~/
       - run:
           name: Test plotly bundles against es6
           command: npm run no-es6-dist
       - run:
           name: Test plotly bundles againt duplicate keys
-          environment:
-            NODE_OPTIONS: --max_old_space_size=4096
```

Comment:
What was this option about, and why are we removing it?

Reply:
It used to be there as a hack to help the `duplicate-key` test run.
It is no longer needed.